From d9f25549356cc21fbba76fd6e1a595309afc02f7 Mon Sep 17 00:00:00 2001
From: Cedric Poignet <cedric.poignet@intel.com>
Date: Thu, 10 Oct 2013 14:05:41 +0200
Subject: [PORT FROM MAIN][BT] Clean False Error Logs

BZ: 143268

Clean or lower trace level of following logs:
E bt_hwcfg: set_param: parameter: * value:

Change-Id: I7b2e8492e7da3786971218c1c0ab33ac7d9ca870
Orig-Change-Id: Ia96a06fb7ef1bfaabe711ffca79ef01ca1bbe36b
Category: aosp improvement
Domain: CWS.BT-Common
Origin: internal
Upstream-Candidate: yes
Signed-off-by: Cedric Poignet <cedric.poignet@intel.com>
Reviewed-on: http://android.intel.com:8080/137563
Reviewed-by: Combes, Patrick <patrick.combes@intel.com>
Tested-by: Lestienne, ChristopheX <christophex.lestienne@intel.com>
---
 src/hardware.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/hardware.c b/src/hardware.c
index 212789c..9f6df5e 100644
--- a/src/hardware.c
+++ b/src/hardware.c
@@ -1482,7 +1482,7 @@ static inline int set_param(char *p_name, char *p_value, int param, int size, \
                             const char *param_name[], uint8_t *bt_param)
 {
     int i;
-    ALOGE( "%s: parameter: %s value: %s", __func__, p_name, p_value);
+    ALOGI( "%s: parameter: %s value: %s", __func__, p_name, p_value);
 
     for (i = 0; i < size; i++) {
          if (strcmp(param_name[i], p_name) == 0) {
-- 
1.7.4.1

